server/src/controllers/prompt.controller.ts

```diff
@@ -48,3 +48,23 @@ export const promptReplaceText = async (req: Request, res: Response) => {
         }
     }
 }
+
+export const promptDelete = async (req: Request, res: Response) => {
+    const { promptId } = req.body;
+    try {
+        const existingPrompt = await PromptModel.findOne({ "_id": promptId });
```

Comment:
for finding things on MongoDB by id, there is a method `PromptModel.findById` which you can use.